h2o-core/src/main/java/water/api/Handler2.java

```diff
@@ -0,0 +1,26 @@
+package water.api;
+
+/**
+ * Common interface for <s>all</s> some REST endpoint handlers.
+ *
+ * @param <IS> input schema class
+ * @param <OS> output schema class
+ */
+public abstract class Handler2<IS extends Schema, OS extends Schema> extends Handler {
```

Comment:
Personally, i do not like naming convention based on numbers - it does not add any value and meaning of 2 is ambiguous - is it version? is it wrapper around Handler? Does '2' stand for 'to' ?